shoulder/model/access_mechanism.py

```diff
@@ -48,9 +41,3 @@ def is_write(self):
     def is_valid(self):
         """ Returns true if the access mechanism is valid """
         return
-
-    @abc.abstractmethod
-    def binary_encoded(self):
-        """ Get the binary instruction encoding that performs an access """
-        """ to/from a register using this access mechanism """
-        return
```

Comment:
Why remove the `instruction_mnemonic` and `binary_encoded` variables from the access_mechanism model? Since we are using both in the generated accessors, shouldn't these be kept?

Reply:
I originally intended the ```instruction_mnemonic``` field as a psuedo-id for the access mechanism, but then nothing ended up using it after we started storing the access mechanisms in a dictionary keyed by the mechanism's name. I was thinking it was dead code at this point since no generators, parsers, transforms, or filters use that field. Is there a reason to put it back I'm not thinking of?

I moved ```binary_encoded``` off of the top-level access mechanism because I'm no longer 100% sure it will be possible to implement that for *all* of the access mechanisms. I also have a feeling that calculating the binary encoded value is going to get more complicated once we start parsing instructions from the XML, so I have it gut feeling it will eventually be moved into it's own separate component.

Reply:
Yeah, that's fine with the `instruction_mnemonic`. The info is redundant since it's stored in the dictionary so I'm fine with ditching it.

After you saying that the 32bit stuff is state dependent, I'm fine leaving off `binary_encoded` from the abstract. I think it still has value for the other mechanisms, but I'm fine leaving that on a case by case basis.